browser/ui/views/split_view/split_view_location_bar.cc

```diff
@@ -36,18 +36,17 @@
 #include "ui/views/widget/widget_delegate.h"
 
 SplitViewLocationBar::SplitViewLocationBar(PrefService* prefs,
-                                           SplitView* split_view)
+                                           views::View* parent_web_view)
     : prefs_(prefs),
-      split_view_(split_view),
       location_bar_model_delegate_(
           std::make_unique<SplitViewLocationBarModelDelegate>()),
       location_bar_model_(std::make_unique<LocationBarModelImpl>(
           location_bar_model_delegate_.get(),
           content::kMaxURLDisplayChars)) {
   set_owned_by_client(OwnedByClientPassKey());
 
-  if (split_view_) {
-    view_observation_.Observe(split_view_->secondary_contents_container());
+  if (parent_web_view) {
+    view_observation_.Observe(parent_web_view);
   } else {
     CHECK_IS_TEST();
   }
```

Comment:
I think we should just not pass the `parent_web_view` in here, and handle this solely from `SetParentWebView`